python/paddle/framework/framework.py

```diff
@@ -116,3 +116,24 @@ def set_grad_enabled(mode):
             tracer._has_grad = prev_mode
     else:
         yield
+
+
+def is_grad_enabled():
+    """
+    Returns currently dygraph gradient calculation mode which is enabled.
+
+    Examples:
+        .. code-block:: python
+            
+            import paddle
+            
+            paddle.is_grad_enabled() # True
```

Comment:
You could add comment "Dygraph gradient calculation mode is enabled by default."

Reply:
done